src/views/Record.vue

```diff
@@ -0,0 +1,167 @@
+<template>
```

Comment:
I can't tell if there were changes made when moving to this new location... however, I either way think the old version should be deleted right?

Reply:
Oh, good call. I forgot to delete the old version.